src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1544,6 +1544,31 @@ elementFormDefault="qualified">
     <xs:element name="AnalysisLevel" type="msb:StringPropertyType" substitutionGroup="msb:Property"/>
     <xs:element name="InstallUrl" type="msb:StringPropertyType" substitutionGroup="msb:Property"/>
     <xs:element name="IsCodeSharingProject" type="msb:boolean" substitutionGroup="msb:Property"/>
+    <xs:element name="IsPackable" type="msb:boolean" substitutionGroup="msb:Property">
+        <xs:annotation>
+            <xs:documentation><!-- _locID_text="IsPackable" _locComment="" -->Indicates whether the project can be used to create a NuGet package.</xs:documentation>
+        </xs:annotation>
+    </xs:element>
+    <xs:element name="EnableNETAnalyzers" type="msb:boolean" substitutionGroup="msb:Property">
+        <xs:annotation>
+            <xs:documentation><!-- _locID_text="EnableNETAnalyzers" _locComment="" -->Indicates whether the CAxxxx are enabled. They are enabled by default only in .NET 5.0 and later versions.</xs:documentation>
```

Comment:
@mikadumont Can you weigh in on this?

Reply:
I think I dropped a word when I wrote this.

```suggestion
            <xs:documentation><!-- _locID_text="EnableNETAnalyzers" _locComment="" -->Indicates whether the CAxxxx analyzers are enabled. They are enabled by default only in .NET 5.0 and later versions.</xs:documentation>
```